### PR TITLE
Finished endpoint for list ranking matrices

### PR DESCRIPTION
**Pull request by dsvalenciah**

### CODE REVIEW

**Comment by odarbelaeze** [comment]
[Construir un endpoint que liste las matrices de ranking](https://trello.com/c/9YHhfHZI/2-construir-un-endpoint-que-liste-las-matrices-de-ranking)

File: app.py [hunk 1]
```diff
@@ -14,6 +13,30 @@
 async def start(request):
     return text("pong")
 
+@app.route("/matrix", methods=["GET", "POST"])
```
Comment (odarbelaeze) [inline app.py]: Let's call this route `/ranking`, and let `matrix` refer to a term document matrix.

File: app.py [hunk 2]
```diff
@@ -14,6 +13,30 @@
...
+@requires_db
+async def matrix(db, request):
```
Comment (odarbelaeze) [inline app.py]: Lets call this one `list_ranking_matrices(db, request)` and limit the endpoint to `GET` requests.

File: app.py [hunk 3]
```diff
@@ -14,6 +13,30 @@
...
+    if request.method == "GET":
```
Comment (odarbelaeze) [inline app.py]: Lets stick to 1 function per HTTP method, to avoid this `if request.method....`

File: app.py [hunk 4]
```diff
@@ -14,6 +13,30 @@
...
+                "build_options": matrix.build_options,
+                "ranking_matrix_path": matrix.ranking_matrix_path
+            }
+            for matrix in list_ranking_matrix(db, count=10)
+        ]
+        return json(ranking_matrices)
+    elif request.method == "POST":
+        return json({"ping": "pong"})
+
+def list_ranking_matrix(db, count):
```
Comment (odarbelaeze) [inline app.py]: We can use plurals here, `list_ranking_matrices_from_db`.... we'll get rid of the `from_db` part as the design evolve.

**Comment by dsvalenciah** [comment]
Simply, execute app.py an go to /ranking route

**Pull request merged**